mindsdb/libs/helpers/general_helpers.py

```diff
@@ -132,8 +133,8 @@ def get_value_bucket(value, buckets, col_stats, hmd=None):
 
     elif col_stats['data_subtype'] in (DATA_SUBTYPES.BINARY, DATA_SUBTYPES.INT, DATA_SUBTYPES.FLOAT):
         bucket = closest(buckets, value)
-    elif col_stats['data_subtype'] in (DATA_SUBTYPES.IMAGE) and hmd is not None:
-        bucket = hmd['bucketing_algorithms'][col_name].predict(np.array(imagehash.phash(Image.open(value)).reshape(1, -1)))[0]
+    # elif col_stats['data_subtype'] in (DATA_SUBTYPES.IMAGE) and hmd is not None:
```

Comment:
Why is this commented out ?

Reply:
Its broken. `col_name` is not defined in this function

Reply:
On another thought, I will uncomment it and make an issue about it

Reply:
https://github.com/mindsdb/mindsdb/issues/485